src/routes/api/users/users_route.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import validate from '../../../middlewares/validator';
+import auth from './auth';
+
+
+const users = express.Router();
+
+users.post('/users', validate.userSignup, auth);
+
+export default users;
```

Comment:
The default export should match the filename. Also, use camel case for filenames.

Reply:
Done